### ISSUE TITLE
Differences from other grunt/casper plugins

### ISSUE THREAD

**Issue by geekdave** [colinwren/grunt-ghost#17]
Hey there, 

Thanks for your work on this plugin.  Just wondering if you can highlight any differences between yours and the other grunt/casper plugins.  Namely:
- https://github.com/iamchrismiller/grunt-casper
- https://github.com/ronaldlokers/grunt-casperjs

All seem to be recently-maintained.  And you listed the last one as one of your influences.  Just wondering what prompted you to create your own.  I'm getting started with Casper and trying to figure out which one to use.

Thanks!
Dave

**Comment by colinwren** [comment]
I created `grunt-ghost` because I had tried `grunt-casperjs` and it had several major problems.  It spawned a casperjs process  for each file instead of running casper once with all the files, it would also log the casper output when it was finished instead of streaming it to `stdout`, it wouldn't fail properly when the tests failed, and the documentation was also pretty much non-existent. 

To fix all of these problems the task needed to be rewritten and thorough documentation of all options needed to be written.  Rather than make a PR to `grunt-capserjs` completely rewriting the task and creating documentation and having to wait for the maintainers response (which at the time he hadn't done anything for months) I figured it would be easier to make my own plugin. 

I haven't looked into `grunt-casperjs` for a while but I suspect some of the problems I mentioned still exist, I remember trying `grunt-casper` and it didn't work at all.  I think `grunt-ghost` would be your best bet because of the problems with `grunt-capserjs` and because of the documentation.

Hope that was helpful, let me know if you have any more questions.
